api/routes.go

```diff
@@ -81,6 +81,9 @@ func (api *API) buildHTTPRoutes() {
 	api.staticRouter.GET("/stripe/prices", api.noAuth(api.stripePricesGET))
 
 	api.staticRouter.GET("/.well-known/jwks.json", api.noAuth(api.wellKnownJWKSGET))
+
+	// Internal endpoint. NEVER EXPOSE THESE!
```

Comment:
wouldn't yell, always kind of weird to see that in a codebase imo

Reply:
Hm, ok. I'm used to seeing all-caps notices in codebases. I don't really consider it yelling - this is not a conversation.